testUtils/httpSpanBuilder.js

```diff
@@ -21,6 +21,10 @@ export class HttpSpanBuilder {
     uri: `${HttpSpanBuilder.DEFAULT_HOST}/`,
   };
 
+  static getDefaultData = data => {
+    return JSON.parse(JSON.stringify(data));
+  };
```

Comment:
Why would we need that? Doesn't it keep the value same as before?

Reply:
Yes it's cloning it

Reply:
Isn't there something like `copy`?